twitchio/ext/routines/__init__.py

```diff
@@ -319,7 +322,18 @@ def completed_iterations(self) -> int:
     def remaining_iterations(self) -> Optional[int]:
         """A count of remaining iterations."""
         return self._remaining_iterations
-
+    
+    @property
+    def time_till_execution(self) -> datetime.timedelta:
```

Comment:
See review comment on this one.

Reply:
Actually, I'm going back through this and it sort of clicked. The naming of the property could be confusing. 

This does expose a `next_event_time`. As an aside, that should probably be marked as private. 

How would you feel about something like `time_until_next_event`? While the docstring and typing should be pretty clear, I figure the naming could also be improved to emphasize that this is a duration estimate.